src/components/atoms/SurveyImg.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import check from '../../assets/images/check.svg';
+
+interface SurveyImgProps {
+  isCheck: boolean;
+  src: string;
+  sort: 'genre' | 'movie';
+}
+
+const SurveyImg = ({ isCheck, src, sort }: SurveyImgProps) => {
+  return (
+    <SurveyImgContainer $sort={sort}>
+      {isCheck && <Background />}
+      {isCheck && <CheckImg src={check} />}
+      <SurveyImage src={src} />
+    </SurveyImgContainer>
+  );
+};
+
+export default SurveyImg;
+
+const SurveyImgContainer = styled.div<{ $sort: SurveyImgProps['sort'] }>`
+  position: relative;
+
+  width: ${({ $sort }) => {
+    if ($sort === 'genre') {
+      return '86px';
+    } else if ($sort === 'movie') {
+      return '88px';
+    }
```

Comment:
if else 도 좋긴 한데, 어차피 $sort 는 'genre' | 'movie' 의 enum 타입이니까 삼항 연산자로 줄여도 무방하긴 해

```suggestion
    return $sort === 'genre' ? '86px' : '88px';
```

Reply:
이부분은 나중에 genre나 movie 말고 다른거 추가되는경우를 고려해서 if문을 사용해보았습니다. 
또한 삼항연산자를 사용하면 'movie' 라는것이 눈에 잘 안들어올 수 도있다고 생각했는데 괜찮을까요??

Reply:
장단점일듯?
확장성을 고려하면 if 가 나을거고
불필요하게 긴 코드가 싫고 가독성을 높이고 싶다면 삼항연산자가 나을거고?
이건 선택의 문제일듯싶다